api/global.json

```diff
@@ -1,7 +0,0 @@
-{
-  "sdk": {
-    "version": "9.0.0",
```

Comment:
I can't apply a suggestion to a deleted line, but try this:    "version": "9.x.x",